### ISSUE TITLE
Importing Mask RCNN - OnnxImportException: Unknown type RoiAlign

### ISSUE THREAD

**Issue by ghost** [Unity-Technologies/barracuda-release#177]
I exported a (resnet) model from pytorch to ONNX. When importing it the following exception is thrown:

```
Asset import failed, "Assets/Assets/NNModels/sketchseg.onnx" > OnnxImportException: Unknown type RoiAlign encountered while parsing layer 2718.

Unity.Barracuda.ONNX.ONNXModelConverter.Err (Unity.Barracuda.Model model, System.String layerName, System.String message, System.String extendedMessage, System.String debugMessage) (at Library/PackageCache/com.unity.barracuda@76077c0b2d/Barracuda/Runtime/ONNX/ONNXModelConverter.cs:3298)
Unity.Barracuda.ONNX.ONNXModelConverter.ConvertOnnxModel (Onnx.ModelProto onnxModel) (at Library/PackageCache/com.unity.barracuda@76077c0b2d/Barracuda/Runtime/ONNX/ONNXModelConverter.cs:2818)
Unity.Barracuda.ONNX.ONNXModelConverter.Convert (Google.Protobuf.CodedInputStream inputStream) (at Library/PackageCache/com.unity.barracuda@76077c0b2d/Barracuda/Runtime/ONNX/ONNXModelConverter.cs:155)
Unity.Barracuda.ONNX.ONNXModelConverter.Convert (System.String filePath) (at Library/PackageCache/com.unity.barracuda@76077c0b2d/Barracuda/Runtime/ONNX/ONNXModelConverter.cs:83)
Unity.Barracuda.ONNXModelImporter.OnImportAsset (UnityEditor.AssetImporters.AssetImportContext ctx) (at Library/PackageCache/com.unity.barracuda@76077c0b2d/Barracuda/Editor/ONNXModelImporter.cs:58)
UnityEditor.AssetImporters.ScriptedImporter.GenerateAssetData (UnityEditor.AssetImporters.AssetImportContext ctx) (at <7587a42c8322471cbc42b27c9b8eab3c>:0)
```

The model is exported the following way:
```
model.load_state_dict(torch.load("segmentation.pth"))
dummy_input = torch.randn(1, 1, 400, 400, requires_grad=True)  # replace with real image resolution?
torch.onnx.export(model, dummy_input, "sketchseg.onnx", verbose=True, do_constant_folding=True,
                  opset_version=11)
```

**Comment by AlexRibard** [comment]
@AnimaNoProject sadly RoiAlign is not supported atm.
We do have plans on adding RoiAlign however, but it might take some time.
I'd suggest changing architecture in the meantime to not use RoiAlign.
(https://towardsdatascience.com/implementing-a-resnet-model-from-scratch-971be7193718)
Could you provide your onnx model? It will help us validate that it is importing once RoiAlign is implemented on our end

**Comment by AlexRibard** [comment]
(closing for now as the task of supporting RoiAlign is already in our system. I'll notify you when the support is out)

**Comment by ghost** [comment]
Hi Thank you for letting me know, how can I share my model with you?
Is there a somewhat timeframe to it (like within one year)?

**Comment by AlexRibard** [comment]
* for the model you can share it via google drive or other cloud based storage/via a archive/email us directly at barracuda-support@unity3d.com
* we expect to land support for this around August
